packages/app-store/routing-forms/components/RoutingNavBar.tsx

```diff
@@ -11,7 +11,7 @@ export default function RoutingNavBar({
 }) {
   const tabs = [
     {
-      name: "Form",
+      name: "Forms",
```

Comment:
I am not sure why we are calling it forms. It is a single Form. I am not sure if it's intentional in figma